docs/samples/browser-plugin-meetings/app.js

```diff
@@ -652,7 +655,13 @@ const toggleSourcesQualityStatus = document.querySelector('#ts-sending-quality-s
 const toggleSourcesMeetingLevel = document.querySelector('#ts-sending-qualities-list');
 const toggleBnrBtn = document.querySelector('#ts-toggle-BNR');
 
-let bnrEnabled = false;
+const loadCameraBtn = document.querySelector('#ts-load-camera');
+const toggleVbgBtn = document.querySelector('#ts-enable-VBG');
+const loadMicrophoneBtn = document.querySelector('#ts-load-mic');
+const toggleBNRBtn = document.querySelector('#ts-enable-BNR');
+const publishShareBtn = document.getElementById('ts-publish-screenshare');
```

Comment:
could you use `document.querySelector('id-name')` rather than `document.getElementById()`. so we can follow the same pattern.

Reply:
Done